man/tpm2_contextload.1.md

```diff
@@ -0,0 +1,53 @@
+% tpm2_contextload(1) tpm2-tools | General Commands Manual
+
+# NAME
+
+**tpm2_contextload**(1) - Loads the session into the TPM, so that it can be
+referenced using the TPM session handle.
+
+# SYNOPSIS
+
+**tpm2_contextload** [*OPTIONS*] [*ARGUMENT*]
+
+# DESCRIPTION
+
+**tpm2_contextload**(1) - Loads the session into the TPM, so that it can be
+referenced using the TPM session handle.
+
+The tool returns the TPM handle the session is currently loaded on, and that
+handle value can be used directly when communicating directly with the TPM.
+
+**NOTE**: The session context file will not be usable on subsequent calls. As
+all tools invoke contextload/contextsave before/after using the session, the
+contextload call for subsequent commands will fail as the session is loaded
+already.
```

Comment:
I did look at this sometime back, but was having a lot more difficulties getting the contextsave working compared to the contextload.

My understanding was that it required changes at the tools library level, since the tpm2_session.c functions that had part of the behaviour to do contextsave seemed to have additional, undesired side-effects.

I'll take another look at it and will try to come up with that as well (provided that it still makes sense to keep this tool pair around, based on my comment below).